### PR TITLE
Database upgrade logging

### PR DESCRIPTION
**Pull request by manadart**
The pertinent change for this patch is to add some info-level logging to the database upgrade worker, so that its output resembles the upgrade-steps worker. This will better illustrate what is happening during upgrades.

In addition, there is some renaming to replace the usage of "master" with "primary" in the same code modules, to reflect the company inclusiveness initiatives.

## QA steps

- Bootstrap a 2.8 controller and enable HA.
- With this patch, run `juju upgrade-controller --build-agent`.
- Observe the controller logs for the `upgradedatabase` worker. We should see something like this:
```
machine-1: 10:15:36 INFO juju.worker.upgradedatabase waiting for database upgrade on mongodb primary
machine-1: 10:15:36 INFO juju.worker.upgradedatabase finished waiting - database upgrade steps completed on mongodb primary
machine-0: 10:15:55 INFO juju.worker.upgradedatabase waiting for database upgrade on mongodb primary
machine-0: 10:15:55 INFO juju.worker.upgradedatabase finished waiting - database upgrade steps completed on mongodb primary
machine-2: 10:15:56 INFO juju.worker.upgradedatabase running database upgrade for 2.9.2.1 on mongodb primary
...
```

## Documentation changes

None.

## Bug reference

N/A

### CODE REVIEW

File: worker/upgradesteps/worker.go [hunk 1]
```diff
@@ -27,26 +27,29 @@ import (
 	"github.com/juju/juju/wrench"
 )
 
-var logger = loggo.GetLogger("juju.worker.upgradesteps")
+var logger = loggo.GetLogger("juju.worker.upgradeSteps")
```
Comment (SimonRichardson) [inline worker/upgradesteps/worker.go:30]: nit: this should be lowercase to reflect the package name.

**Review by SimonRichardson — APPROVED** [review]

File: worker/upgradesteps/worker.go (hunk 1 above)
Comment (manadart) [reply worker/upgradesteps/worker.go]: Good catch. Fixed.

**Comment by manadart** [comment]
[Increase log output for database-upgrade worker](https://trello.com/c/MPq40k8C/3665-increase-log-output-for-database-upgrade-worker)

**Comment by manadart** [comment]
$$merge$$

**Pull request merged**